tools/functional-tester/etcd-tester/cluster.go

```diff
@@ -36,10 +36,12 @@ const (
 type cluster struct {
 	v2Only bool // to be deprecated
 
-	datadir              string
+	agentEndpoints       []string
```

Comment:
I removed this in the last clean up. Is there a reason it needs to be in this struct?